packages/feedback/src/types/index.ts

```diff
@@ -299,6 +303,14 @@ export interface FeedbackTheme {
    * Border styles for form inputs when focused
    */
   inputOutlineFocus: string;
+  /**
+   * Border radius for dialog
+   */
+  formBorderRadius: string;
+  /**
+   * Border radius for form inputs
+   */
+  formContentBorderRadius: string;
```

Comment:
not 100% sure but what about maybe `inputBorderRadius` or something like this?

Reply:
we considered `inputBorderRadius`, but this applies to the input and the buttons in the form (cancel & submit) so we wanted it to be more clear